### ISSUE TITLE
feat(ui): add relevant contextual info/KPI in entity list and highlight relevant information in entity details

### ISSUE THREAD

**Issue by ghost** [LuatixHQ/opencti#6645]
## Use case
Currently the most (more then 50%) of the buttons in the sidebar just trigger a view of all entities filtered by entity type.
A real knowledge improvement concerning the specific entity is not recognizableable.
This leads to a high complexity for looking up specific entities. Most of the time I just use data / entities or data / relations if I'm looking up something specific. If I need visualizations I use a Dashboard.

It would be more beneficial if there were at least some "Overview sites" for the entities instead of plain filtered lists.

E.g. If I go to Intrusion Sets I don't just see the cards (which are by the way even worser then a plain list, looks cool but absolute no benefit) I would have an overview with important KPIs and e.g. a List with the newest Intrustion Sets, most active Intrusion Sets, latest active,...
This also applies to Malwares in the exact same way.

Concerning Indicators or Observables I would also prefer to see an Overview with latest, total no, new ones in the last 24h, a list with the ones with a high risk score (domains and IPs),...

**All of the above mentioned things would / should be possible with the current dashboards**

Further the "Details" pages of a specific entity are mostly not perfect to get the information that are really counting. Overview is good. What else wood be needed would be relations, Risk Scoring (Currently hidden in a highly complex view about confidence and reliability that doens't provide a reliability at all (A real reliability is made of the no of confirmations, this no of confirmations is in the reality not made of analysts that confirm something manually but by the no of sources that are imported by connectors an).
The labels of the entity are also hidden in this view but would be more appropriate at the overview level.

**Summing up things: Also in this place a Dashboard based approach would be beneficial.**

### Proposed Solution
By Enhancing the current Dashboards with additional Widgets that are currently hardcoded in the details pages (Overview, Reliability, Matrix, Knowledge Graph View),.. a very customizable UI could be created that would be capable to address anyones needs and can lead to real information gains.
